css/styles.css

```diff
@@ -1234,7 +1234,7 @@ input[type='checkbox'] {
 .slider-2 {
 	padding-top: 6.875rem;
 	padding-bottom: 6.875rem;
-	background-color: #2f2c3d;
+	background-color: #9baec8;
```

Comment:
The text on this background is a little hard to read as the new background color blends in with the text color. Also, it is too close to grey. What do you think?

I think you could use the material color palette and choose three or more colors that are close to each other.